Update repository restrictions (#2366)
theforeman/foreman-documentation: guides/common/modules/con_composite-content-views-overview.adoc

You can also select the *Auto Publish* option when creating a Composite Content View, and then the Composite Content View is automatically republished when a Content View it includes is republished.

> .Repository Restrictions

Reviewer: ```suggestion
.Repository restrictions
```

Reviewer: Rest LGTM!

> You cannot include more than one of each container repository in Composite Content Views.

Reviewer: ```suggestion
Each container repository cannot be included more than once in a Composite Content View.
```
An alternative if you like.
